etl/.env.example

```diff
@@ -10,3 +10,6 @@ POSTGRES_SOURCE_PREOP_SCHEMA=preop_schema
 POSTGRES_SOURCE_INTRAOP_SCHEMA=intraop_schema
 POSTGRES_SOURCE_POSTOP_SCHEMA=postop_schema
 BASE_PATH=pypasar/db/sql/postgres/
+
+### Replace with your own source_vocabulary_id from the source_to_concept_map table ###
+SOURCE_VOCAB_ID=STCM_SOURCE_VACAB_ID
```

Comment:
`SOURCE_VOCAB_ID` used?

Reply:
The `SOURCE_VOCAB_ID` is used when mapping with the STCM table to retrieve the concept IDs. For example, assume that:
SOURCE_VOCAB_ID = RACE_VOCAB_ID
```sql
JOIN {OMOP_SCHEMA}.source_to_concept_map AS stcm
WHERE stcm.source_vocabulary_id = '{SOURCE_VOCAB_ID}'
```

Reply:
1. I dont see `SOURCE_VOCAB_ID` being referenced in the code. I see only this at line 48
![image](https://github.com/user-attachments/assets/4d36bbae-a11c-4d7c-9c04-f0ef66274e90)

2. Since there are multiple vocabularies for various tables. When we run the ETL for all the tables this wouldnt scale. Therefore its fine if the source vocabulary like `SG_PASAR_RACE` vale be hardcoded in the sql

Reply:
1. Just to clarify, SOURCE_VOCAB_ID is just an example; the actual value I’m using is RACE_VOCAB_ID.
2. I appreciate your insight on this, so I’ll go ahead and close this PR.